src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,131 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User expected = new User("tarsan@Gmail.com", "123456", 30);
+        User actual = registrationService.register(expected);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_userLoginIsNull_notOk() {
+        User user = new User(null, "123456", 30);
+        var exception = Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(user));
+
+        var expected = "User login cannot be null";
```

Comment:
If you change exception message in main code your test will fail, so don't need to check the message. 
Or, in real projects messages could be stored in constants.